inception-project/inception
# Feature/266 support import export in logging module

**What's in the PR**
* Do not skip events without document association on export
* Properly restore events without document association on import

**How to test manually**
* Export a project and check that the exported log contains event such as   `AfterProjectCreatedEvent`
* Import the project again and check the document column of the event has a `-1`

**Automatic testing**

**Documentation**
